[Accordion] Update docs (#175)
radix-ui/website: data/primitives/components/accordion/0.0.16.mdx

<Highlights
  features={[
    'Full keyboard navigation.',

>     'Can expand only one or multiple items.',

Author: I think perhaps

```suggestion
    'Can expand one or multiple items.',
```

is even clearer

>     'Can expand only one or multiple items.',

Reviewer: Even better 👍

      typeSimple: 'enum',
      description: (
        <span>

>           Determines whether a single item or multiple items can be
>           opened at the same time.

Reviewer: ```suggestion
          Determines whether single or multiple items can be
          opened at the same time.
```

And similar here perhaps?

Author: ```suggestion
          Determines whether one or multiple items can be
          opened at the same time.
```

?

Reviewer: 👍
